index.html

```diff
@@ -18,22 +18,125 @@
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio.png" alt="Snapshoot-1" />
+        <p class="project-title">Tonic</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (1).png" alt="Snapshoot-1" />
+        <p class="project-title">Multi-Post Stories</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (2).png" alt="Snapshoot-1" />
+        <p class="project-title">Tonic</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (3).png" alt="Snapshoot-1" />
+        <p class="project-title">Multi-Post Stories</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="about-section">
+        <p class="about-me-text">About Myself</p>
```

Comment:
- [ ] As I mentioned before I would like to recommend you use `<hx>` elements instead `<p>` tags for titles and subtitles, because it is a good practice and semantically correct :+1: